### PR TITLE
Made edit form for info

### LINKED ISSUE #444: feat: Allow users to edit their info
**Is your feature request related to a problem? Please describe.**
Users can't edit their own information in https://datateknologerna.org/members/info/
This leads to users info being out of date or manual intervention being required by the board to update the information. 

**Describe the solution you'd like**
Make the user info editable or create a separate edit form. 

**Additional context**

### PR DESCRIPTION
**Pull request by Theoabw**

### CODE REVIEW

File: members/forms.py [hunk 1]
```diff
@@ -218,4 +218,18 @@ class Meta:
...
+class MemberEditForm(forms.ModelForm):
+    email = forms.EmailField(required=True)
+    class Meta:
+        model = Member
+        fields = ['email', 'first_name', 'last_name', 'phone', 'address', 'zip_code', 'city', 'country']
+
+    def clean_email(self):
+        email = self.cleaned_data.get('email')
+        if not email:
+            raise forms.ValidationError("This field is required.")
```
Comment (jaspnas) [inline members/forms.py]: I think this should be validated by the EmailField being set as required

File: members/views.py [hunk 2]
```diff
@@ -18,7 +18,8 @@
 from django.views import View
 
 from core.utils import validate_captcha
-from members.forms import SignUpForm, AlumniSignUpForm, FunctionaryForm
+from members.forms import SignUpForm, AlumniSignUpForm, FunctionaryForm, MemberEditForm
+from . import forms
```
Comment (jaspnas) [inline members/views.py]: Seems like an  unused import, also could be confused with how we usually import the django forms module

File: members/forms.py [hunk 3]
```diff
@@ -218,4 +218,18 @@ class Meta:
 
 class FunctionaryFilterForm(forms.Form):
     year = forms.IntegerField(required=False)
-    functionary_role = forms.CharField(required=False)
\ No newline at end of file
+    functionary_role = forms.CharField(required=False)
+
+
+class MemberEditForm(forms.ModelForm):
```
Comment (jaspnas) [inline members/forms.py:252]: Should maybe change the MemberupdateForm to AdminMemberUpdateForm for clarity

File: members/forms.py (hunk 1 above)
Comment (theoabw) [reply members/forms.py]: Yeah, I threw the whole email validation method in the trash bin.

File: members/forms.py [hunk 4]
```diff
@@ -247,3 +247,11 @@ class Meta:
 class FunctionaryFilterForm(forms.Form):
     year = forms.IntegerField(required=False)
     functionary_role = forms.CharField(required=False)
+
+
+class MemberEditForm(forms.ModelForm):
+    email = forms.EmailField(required=True)
```
Comment (jaspnas) [inline members/forms.py]: Do we want users to be able to edit their email address?

File: members/forms.py [hunk 5]
```diff
@@ -247,3 +247,11 @@ class Meta:
...
+
+    class Meta:
+        model = Member
+        fields = ['email', 'first_name', 'last_name', 'phone', 'address', 'zip_code', 'city', 'country']
```
Comment (jaspnas) [inline members/forms.py]: The form allows the user to set their first- and last name to blank, this should be fixed in the model by setting blank to false for it.

**Review by jaspnas — DISMISSED** [review]
Please resolve the earlier comments

**Review by jaspnas — APPROVED** [review]

**Pull request merged**